sessions/css-animations/toast-animations/css/styles.css

```diff
@@ -5,37 +5,15 @@
 }
 
 body {
-  background-color: #e4e8f0;
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
-    Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
-  position: relative;
```

Comment:
There seems to be no element with the property value 'relative' anymore after the deletion of this. I'm not sure if that's intentional? `position: absolute` on the respective elements still works, but mostly because they'll grab onto the initial containing block, which is the viewport. It might not make a difference in this case, but it's not exactly self-explanatory. If we're teaching students that relative/absolute is a pair that needs one another, I'm not sure if we should just forego either.

Reply:
Since it is not needed I decided to remove it. We can put it back in if that fits better with what we want to teach.

I am not sure about that they need each other, because in this case, they don't.